src/components/Employee/Deductions/IncludeDeductionsForm/IncludeDeductionsFormEmptyState.tsx

```diff
@@ -0,0 +1,103 @@
+import { useTranslation } from 'react-i18next'
+import { z } from 'zod'
+import { FormProvider, useForm, type SubmitHandler } from 'react-hook-form'
+import { zodResolver } from '@hookform/resolvers/zod'
+import styles from './IncludeDeductionsFormEmptyState.module.scss'
+import {
+  BaseComponent,
+  type BaseComponentInterface,
+  type CommonComponentInterface,
+  useBase,
+} from '@/components/Base'
+import { Form } from '@/components/Common/Form'
+import { Grid } from '@/components/Common/Grid/Grid'
+import { ActionsLayout, Flex } from '@/components/Common'
+import { useI18n } from '@/i18n'
+import { componentEvents } from '@/shared/constants'
+import { useComponentContext } from '@/contexts/ComponentAdapter/useComponentContext'
+import { useComponentDictionary } from '@/i18n/I18n'
+import CoinsHandsIcon from '@/assets/icons/coins-hand.svg?react'
+import PlusCircleIcon from '@/assets/icons/plus-circle.svg?react'
+
+const IncludeDeductionsSchema = z.object({ includeDeductions: z.enum(['Yes', 'No']) })
+export type IncludeDeductionsPayload = z.output<typeof IncludeDeductionsSchema>
+
+interface IncludeDeductionsFormProps extends CommonComponentInterface<'Employee.Deductions'> {
+  employeeId: string
+}
+
+export function IncludeDeductionsFormEmptyState(
+  props: IncludeDeductionsFormProps & BaseComponentInterface,
+) {
+  return (
+    <BaseComponent {...props}>
+      <Root {...props}>{props.children}</Root>
+    </BaseComponent>
+  )
+}
+
+function Root({ className, children, dictionary }: IncludeDeductionsFormProps) {
+  const { onEvent } = useBase()
+  const { t } = useTranslation('Employee.Deductions')
+  const Components = useComponentContext()
+
+  useComponentDictionary('Employee.Deductions', dictionary)
+  useI18n('Employee.Deductions')
+
+  const formMethods = useForm<IncludeDeductionsPayload>({
+    resolver: zodResolver(IncludeDeductionsSchema),
+    defaultValues: { includeDeductions: 'No' },
+  })
+
+  const handleAdd = () => {
+    onEvent(componentEvents.EMPLOYEE_DEDUCTION_INCLUDE_YES)
+  }
+
+  const onSubmit: SubmitHandler<IncludeDeductionsPayload> = () => {
+    onEvent(componentEvents.EMPLOYEE_DEDUCTION_INCLUDE_NO)
+  }
+
+  return (
+    <section className={className}>
+      <FormProvider {...formMethods}>
+        <Form onSubmit={formMethods.handleSubmit(onSubmit)}>
+          {children ? (
+            children
+          ) : (
+            <Grid gridTemplateColumns="1fr">
+              <Components.Heading as="h2">{t('pageTitle')}</Components.Heading>
+              <Components.Text variant="supporting">
+                {t('includeDeductionsDescriptionV2')}
+              </Components.Text>
+              <Components.Text weight="bold" size="lg">
+                {t('includeDeductionsSubtitle')}
+              </Components.Text>
+              <Components.Card className={styles.emptyStateContainer}>
```

Comment:
I'd avoid using card here, this is likely just a layout specific container and could be styled here as a one off. Card will get overwritten with component adapters as something more traditionally card like which could be more responsive leaning